release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -57,6 +57,21 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-wildcard-subjects-headless-services"]
+==== Wildcard subject for service serving certificates for headless services
+
+Generating a service service certificate for headless services now includes a wildcard subject in the format of `*.<service.name>.<service.namespace>.svc`. This allows for TLS-protected connections to individual stateful set pods without having to manually generate certificates for these pods.
+
+[IMPORTANT]
+====
+Because the generated certificates contain wildcard subjects for headless services, you must not use the service CA if your client must differentiate between individual pods. In this case, do the following:
+
+* Generate individual TLS certificates using a different CA.
+* Do not accept the service CA as a trusted CA for connections that are directed to individual pods and must not be impersonated by other pods. These connections must be configured to trust the CA that was used to generate the individual TLS certificates.
```

Comment:
- I often find "do not" to be easier to understand than "you must not", which might also help here because "must" appears again in the same sentence. 🤷 

- `s/using/by using/` (for clarification)

- Also, can you rephrase the lead-in sentence? ISG doesn't like "do the following (and "Do the following...Do not accept" is a bit of a contradiction).

  **following** _n_
  Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”

Reply:
@bobfuru Yeah I struggled with that lead-in to the list. I didn't like "do the following" but I did see it a few places in the repo and couldn't think of anything better. But I'm definitely up for finding a better alternative - let me know if you can think of any suggestions! I'll look at this fresh again tomorrow and see if I get a stroke of genius ;)

Reply:
I shortened to just "In this case" for now, but will think on it again

Reply:
I think "In this case" works great in this case! 😁